### PR TITLE
pt-BR: Translate 'how-to-use-util-inspect'

### PR DESCRIPTION
**Pull request by vbertazzo**
Hey everyone,

This PR translates to Brazilian Portuguese the page `how-to-use-util-inspect` from the section `getting-started`.

@glaucia86 @khaosdoctor @nodejs/nodejs-pt

### CODE REVIEW

File: locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md [hunk 1]
```diff
@@ -0,0 +1,88 @@
...
+---
+
+<!-- Node provides a utility function, for debugging purposes, that returns a string representation of an object.  `util.inspect()` can be a true lifesaver while working with properties of large, complex objects. 
+
+Let's provide a basic example. `util.inspect()` can be used on any object - a good demonstration will be one of Node's built-in objects.  Try this in the REPL (type `node` at your command line with no arguments):
+
+     var util = require('util');
+     util.inspect(console); -->
+     
+O Node oferece uma função utilitária, para fins de depuração, que retorna uma representação string de um objeto. `util.inspect ()` pode ser de grande ajuda quando utilizada com propriedades de grandes e complexos objetos.
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md]:
```suggestion
O Node oferece uma função utilitária, para fins de depuração, que retorna uma representação string de um objeto. `util.inspect()` pode ser de grande ajuda quando utilizada com propriedades de objetos que são muito grandes ou complexos.
```

File: locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md [hunk 2]
```diff
@@ -0,0 +1,88 @@
...
+
+The `depth` argument is the number of levels deep into a nested object to recurse - it defaults to 2.  Setting it to `null` will cause it to recurse 'all the way', showing every level.  Compare the (size of) the outputs of these two `util.inspect` statements in the REPL:
+
+     var http = require('http');
+     util.inspect(http, true, 1);
+     util.inspect(http, true, 3); -->
+
+Por exemplo, `util.inspect(myObj, true, 7, true)` inspecionaria `myObj`, mostrando todas as propriedades ocultas e não ocultas até uma profundidade de `7` e coloriria o output. Vamos conferir os argumentos individualmente.
+
+O argumento `depth` é o número de níveis dentro de um objeto aninhado para ser recursivo - o padrão é 2. Configurá-lo para `null` fará com que ele recurse tudo, mostrando todos os níveis. Compare o tamanho dos outputs dessas duas instruções `util.inspect` no REPL:
```
Comment (khaosdoctor) [inline locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md]:
```suggestion
O argumento `depth` é o número de níveis dentro de um objeto aninhado para ser recursivo - o padrão é 2. Configurá-lo para `null` fará com que ele vá até o último nível, mostrando todos os níveis. Compare o tamanho dos outputs dessas duas instruções `util.inspect` no REPL:
```

**Review by khaosdoctor — CHANGES_REQUESTED** [review]
A typo and a few suggestions for better understanding

**Review by khaosdoctor — APPROVED** [review]
LGTM

**Comment by khaosdoctor** [comment]
@Trott @MaledongGit

File: locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md [hunk 3]
```diff
@@ -0,0 +1,88 @@
...
+     var util = require('util');
+     util.inspect(console);
+
+<!-- The output will be:
+
+     '{ log: [Function], info: [Function], warn: [Function], error: [Function], dir: [Function], time: [Function], timeEnd: [Function], trace: [Function], assert: [Function] }'
+     
+This is a listing of all the enumerable properties of the `console` object.  It is also worth noting that `console.dir` is a wrapper around `util.inspect` that uses its default arguments. -->
+
+O output vai ser:
```
Comment (TiagoDanin) [inline locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md]:
```suggestion
O resultado vai ser:
```
or
```suggestion
A saída vai ser:
```

File: locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md [hunk 4]
```diff
@@ -0,0 +1,88 @@
...
+
+     '{ log: [Function], info: [Function], warn: [Function], error: [Function], dir: [Function], time: [Function], timeEnd: [Function], trace: [Function], assert: [Function] }'
+
+Esta é uma listagem de todas as propriedades enumeráveis ​​do objeto `console`. Também vale a pena notar que `console.dir` é um wrapper em torno de ` util.inspect` que utiliza seus argumentos padrão.
```
Comment (TiagoDanin) [inline locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md]:
```suggestion
Esta é uma listagem de todas as propriedades enumeráveis ​​do objeto `console`. Também vale a pena notar que `console.dir` é um wrapper em torno de ` util.inspect` que utiliza seus argumentos padrões.
```

File: locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md [hunk 5]
```diff
@@ -0,0 +1,88 @@
...
+
+`util.inspect` can also be passed several optional arguments, shown here with their defaults:
+
+     util.inspect(object, showHidden=false, depth=2, colorize=true); -->
+
+No REPL, `util.inspect` retornará imediatamente seu output - esse não é o caso normalmente. No contexto de um código Node.js comum em um arquivo, algo deve ser feito com o output. A coisa mais simples a fazer:
+
+     console.log(util.inspect(myObj));
+
+`util.inspect` pode também receber diversos argumentos opcionais, mostrados aqui com seus valores padrão:
```
Comment (TiagoDanin) [inline locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md]:
```suggestion
`util.inspect` pode também receber diversos argumentos opcionais, mostrados aqui com seus valores padrões:
```

File: locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md [hunk 6]
```diff
@@ -0,0 +1,88 @@
...
+<!--
+For example, `util.inspect(myObj, true, 7, true)` would inspect `myObj`, showing all the hidden and non-hidden properties up to a depth of `7` and colorize the output.  Let's go over the arguments individually.
+
+The `depth` argument is the number of levels deep into a nested object to recurse - it defaults to 2.  Setting it to `null` will cause it to recurse 'all the way', showing every level.  Compare the (size of) the outputs of these two `util.inspect` statements in the REPL:
+
+     var http = require('http');
+     util.inspect(http, true, 1);
+     util.inspect(http, true, 3); -->
+
+Por exemplo, `util.inspect(myObj, true, 7, true)` inspecionaria `myObj`, mostrando todas as propriedades ocultas e não ocultas até uma profundidade de `7` e coloriria o output. Vamos conferir os argumentos individualmente.
```
Comment (TiagoDanin) [inline locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md]:
```suggestion
Por exemplo, `util.inspect(myObj, true, 7, true)` inspecionaria `myObj`, mostrando todas as propriedades ocultas e não ocultas até uma profundidade de `7` e coloriria o resultado. Vamos conferir os argumentos individualmente.
```

File: locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md [hunk 7]
```diff
@@ -0,0 +1,88 @@
...
+
+O argumento `depth` representa o número de níveis dentro de um objeto aninhado para ser recursivo - o padrão é 2. Configurá-lo para `null` fará com que ele vá até o último nível, mostrando todos os níveis. Compare o tamanho dos outputs dessas duas instruções `util.inspect` no REPL:
```
Comment (TiagoDanin) [inline locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md]:
```suggestion
O argumento `depth` representa o número de níveis dentro de um objeto aninhado para ser recursivo - o padrão é 2. Configurá-lo para `null` fará com que ele vá até o último nível, mostrando todos os níveis. Compare o tamanho dos resultados dessas duas instruções `util.inspect` no REPL:
```

File: locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md [hunk 8]
```diff
@@ -0,0 +1,88 @@
...
+     var http = require('http');
+     util.inspect(http, true, 1);
+     util.inspect(http, true, 3);
+
+<!-- The optional argument `showHidden` is a boolean that determines whether or not the 'non-enumerable' properties of an object will be displayed - it defaults to `false`, which tends to result in vastly more readable output.  This isn't something a beginner needs to worry about most of the time, but it's worth demonstrating briefly.  Once more, try the following in the REPL:
+
+     var util = require('util');
+     util.inspect(console, true); -->
+
+O argumento opcional `showHidden` é um booleano que determina se as propriedades 'não-enumeráveis' de um objeto serão ou não exibidas - o padrão é `false`, o que tende a resultar em um output muito mais legível. Isso não é algo que um iniciante precise se preocupar na maior parte do tempo, mas vale a pena demonstrar brevemente. Mais uma vez, tente o seguinte no REPL:
```
Comment (TiagoDanin) [inline locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md]:
```suggestion
O argumento opcional `showHidden` é um booleano que determina se as propriedades 'não-enumeráveis' de um objeto serão ou não exibidas - o padrão é `false`, o que tende a resultar em uma saída muito mais legível. Isso não é algo que um iniciante precise se preocupar na maior parte do tempo, mas vale a pena demonstrar brevemente. Mais uma vez, tente o seguinte no REPL:
```

File: locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md [hunk 9]
```diff
@@ -0,0 +1,88 @@
...
+
+     var util = require('util');
+     util.inspect(console, true);
+
+<!-- Finally, the optional argument `colorize` is a boolean that adds ANSI escape codes to the string output. When logged to a terminal window, it should be pretty printed with colors.
+
+     var util = require('util');
+     console.log(util.inspect({a:1, b:"b"}, false,2,true)); -->
+
+Finalmente, o argumento opcional `colorize` é um booleano que adiciona códigos de escape ANSI ao output da string. Quando utilizado em uma janela de terminal, o output deve aparecer colorido.
```
Comment (TiagoDanin) [inline locale/pt-br/knowledge/getting-started/how-to-use-util-inspect.md]:
```suggestion
Finalmente, o argumento opcional `colorize` é um booleano que adiciona códigos de escape ANSI ao resultado da string. Quando utilizado em uma janela de terminal, o output deve aparecer colorido.
```

**Review by TiagoDanin — CHANGES_REQUESTED** [review]
Do not mix Portuguese with Enslish

**Review by TiagoDanin — APPROVED** [review]
LGTM :100:

**Pull request merged**